src/main/java/com/truelayer/java/http/auth/AccessTokenManager.java

```diff
@@ -20,7 +23,8 @@ public class AccessTokenManager implements IAccessTokenManager {
 
     private final ICredentialsCache credentialsCache;
 
-    private final List<String> scopes = singletonList(Constants.Scopes.PAYMENTS);
+    private final List<String> scopes = Collections.unmodifiableList(Arrays.asList(Constants.Scopes.PAYMENTS,
+            Constants.Scopes.RECURRING_PAYMENTS_SWEEPING));
```

Comment:
This is temporary.

We should opt in for scopes as required by the actually used APIs. Setting a catch-all list of scopes here that maps to all the capabilities offered by the library might break existing integration 🤯